databricks_cli/cli.py

```diff
@@ -54,6 +55,7 @@ def cli():
 cli.add_command(workspace_group, name='workspace')
 cli.add_command(jobs_group, name='jobs')
 cli.add_command(clusters_group, name='clusters')
+cli.add_command(cluster_policies_group, name='cluster_policies')
```

Comment:
can we make this `cluster-policies`? That would be consistent with `instance-pools`.

Reply:
Done